src/server/ApiRoutes/Bib.js

```diff
@@ -9,19 +9,24 @@ import { isNyplBnumber } from '../../app/utils/utils';
 import { appendDimensionsToExtent } from '../../app/utils/appendDimensionsToExtent';
 
 const nyplApiClientCall = (query, itemFrom, filterItemsStr = "") => {
-  const queryForItemPage = typeof itemFrom !== 'undefined' ? `?items_size=${itemBatchSize}&items_from=${itemFrom}` : '';
+  const itemRelatedQueries = []
+  if (typeof itemFrom !== 'undefined') itemRelatedQueries.push(`items_size=${itemBatchSize}&items_from=${itemFrom}`)
   let fullQuery;
   if (query.includes('.annotated-marc')) {
     fullQuery = query;
   } else {
-    const itemQuery = (filterItemsStr.length ? `&${filterItemsStr}` : '');
-    fullQuery = `${query}${queryForItemPage}${itemQuery}&merge_checkin_card_items=true`
+    if (filterItemsStr.length) itemRelatedQueries.push(`${filterItemsStr}`)
+    if (itemRelatedQueries.length) {
+      itemRelatedQueries.push('merge_checkin_card_items=true')
+    }
```

Comment:
Why only add `merge_checkin_card_items=true` when there are other "itemRelatedQueries"? Shouldn't we always add it to the array?

Reply:
Yeah I was thinking it that case we don't need any item-related query, but I guess it makes sense to just always have it there. Fixed.